linux/jdk/debian/src/main/packaging/microsoft/11/debian/changelog

```diff
@@ -1,3 +1,9 @@
+msopenjdk-11 (11.0.21-1) STABLE; urgency=medium
+
+  * Microsoft 11.0.21-1 release.
+
+ -- Microsoft Package Maintainers <openjdk@microsoft.com>  Wed, 11 Oct 2023 10:29:35 +0000
```

Comment:
Is that date correct?

Reply:
I verified the date with the Team. The date is correct.